Index: src/main/java/com/IpayUsers/IpayUsers/Domain/Customer_Bank_Account.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.IpayUsers.IpayUsers.Domain;\r\n\r\nimport jakarta.persistence.*;\r\nimport lombok.AllArgsConstructor;\r\nimport lombok.Data;\r\nimport lombok.NoArgsConstructor;\r\n\r\nimport java.util.List;\r\n\r\n@Entity\r\n@NoArgsConstructor\r\n@AllArgsConstructor\r\n@Data\r\npublic class Customer_Bank_Account {\r\n    @Id\r\n    private int Account_Number;\r\n    private String status;\r\n    private String is_default;\r\n\r\n\r\n    @OneToMany(cascade = CascadeType.ALL)\r\n    @JoinColumn(name = \"fk_Account_Number\",referencedColumnName =\"Account_Number\" )\r\n    private List<Customer> customer;\r\n}\r\n
===================================================================
diff --git a/src/main/java/com/IpayUsers/IpayUsers/Domain/Customer_Bank_Account.java b/src/main/java/com/IpayUsers/IpayUsers/Domain/Customer_Bank_Account.java
--- a/src/main/java/com/IpayUsers/IpayUsers/Domain/Customer_Bank_Account.java	
+++ b/src/main/java/com/IpayUsers/IpayUsers/Domain/Customer_Bank_Account.java	
@@ -19,6 +19,6 @@
 
 
     @OneToMany(cascade = CascadeType.ALL)
-    @JoinColumn(name = "fk_Account_Number",referencedColumnName ="Account_Number" )
+    @JoinColumn(name = "fk_Account_number",referencedColumnName ="Account_Number" )
     private List<Customer> customer;
 }
